src/twisted/conch/scripts/ckeygen.py

```diff
@@ -339,23 +368,23 @@ def _saveKey(key, options):
 
     comment = f"{getpass.getuser()}@{socket.gethostname()}"
 
-    filepath.FilePath(options["filename"]).setContent(
+    filepath.FilePath(filename).setContent(
```

Comment:
Huh. Not your bug, but this is such an odd choice to not use the path object here…

```suggestion
    fp = filepath.FilePath(filename)
    fp.setContent(
```

Reply:
Updated